src/BloomBrowserUI/bookEdit/js/bloomImages.js

```diff
@@ -54,38 +54,62 @@ function SetupImage(image) {
     });
 }
 
+function SetButtonModifier(container) {
```

Comment:
As far as I can tell, this GETS the button modifier (at least what you want it to be). It doesn't SET anything.